packages/playwright/src/runner/testServer.ts

```diff
@@ -108,7 +113,6 @@ export class TestServerDispatcher implements TestServerInterface {
 
     this._dispatchEvent = (method, params) => this.transport.sendEvent?.(method, params);
     this._testRunner.on(TestRunnerEvent.TestFilesChanged, testFiles => this._dispatchEvent('testFilesChanged', { testFiles }));
-    this._testRunner.on(TestRunnerEvent.TestPaused, params => this._dispatchEvent('testPaused', { errors: params.errors }));
```

Comment:
I thought this would also remove `TestRunnerEvent.TestPaused`. Otherwise it seems like adding one more channel (to the reporter) without getting rid of the alternative `TestPaused` channel.